src/API.jl

```diff
@@ -487,7 +487,10 @@ function status(ctx::Context, pkgs::Vector{PackageSpec}; mode=PKGMODE_PROJECT)
 end
 
 
-activate() = (Base.ACTIVE_PROJECT[] = nothing)
+function activate()
+    Base.ACTIVE_PROJECT[] = nothing
+    @info("activated home project environment at $(Base.active_project()).")
```

Comment:
```suggestion
    @info("activated environment at $(pathrepr(Base.active_project())).")
```

Reply:
Hmm, `pathrepr` calls `contractuser`: https://github.com/JuliaLang/Pkg.jl/blob/32396f241ac5e957dd078d22e22cee2107f7bdc8/src/Types.jl#L1338 so we should probably use that everywhere instead of `contractuser`. Can be a separate PR though.